### ISSUE TITLE
enable_saml integration testing failure

### ISSUE THREAD

**Issue by cliffano** [shinesolutions/puppet-aem-resources#106]
**Describe the bug**
Integration testing enable_saml is failing when tested against AEM 6.5 (no service pack).

```
^[[mNotice: /Stage[main]/Main/Aem_resources::Enable_saml[Enable SAML authentication]/Aem_node[author: Create Apache Sling Referrer Filter config node]: Dependency Aem_saml[aem_saml] has failures: true^[[0m
```

```
            <tr>
                <td>Location</td>
                <td><img class="cq-LinkChecker cq-LinkChecker--prefix cq-LinkChecker--invalid" src="/libs/cq/linkchecker/resources/linkcheck_o.gif" alt="invalid link: /apps/system/config/com.adobe.granite.auth.saml.SamlAuthenticationHandler.config" title="invalid link: /apps/system/config/com.adobe.granite.auth.saml.SamlAuthenticationHandler.config" border="0">/apps/system/config/com.adobe.granite.auth.saml.SamlAuthenticationHandler.config<img class="cq-LinkChecker cq-LinkChecker--suffix cq-LinkChecker--invalid" src="/libs/cq/linkchecker/resources/linkcheck_c.gif" border="0"></td>
            </tr>
            <tr>
                <td>Parent Location</td>
                <td><a href="/apps/system/config" id="ParentLocation">/apps/system/config</a></td>
            </tr>
            <tr>
                <td>Path</td>
                <td><div id="Path">/apps/system/config/com.adobe.granite.auth.saml.SamlAuthenticationHandler.config</div></td>
            </tr>
            <tr>
                <td>Referer</td>
                <td><div id="Referer"></div></td>
            </tr>
            <tr>
                <td>ChangeLog</td>
                <td><div id="ChangeLog">&lt;pre&gt;&lt;/pre&gt;</div></td>
            </tr>
        </tbody>
    </table>
    <p><img class="cq-LinkChecker cq-LinkChecker--prefix cq-LinkChecker--invalid" src="/libs/cq/linkchecker/resources/linkcheck_o.gif" alt="invalid link: /apps/system/config/com.adobe.granite.auth.saml.SamlAuthenticationHandler.config" title="invalid link: /apps/system/config/com.adobe.granite.auth.saml.SamlAuthenticationHandler.config" border="0">Modified Resource<img class="cq-LinkChecker cq-LinkChecker--suffix cq-LinkChecker--invalid" src="/libs/cq/linkchecker/resources/linkcheck_c.gif" border="0"></p>
    <p><a href="/apps/system/config">Parent of Modified Resource</a></p>
    </body>
</html> (file: /opt/workspace/workspace-aoc/puppet-aem-resources/test/integration/modules/aem_resources/manifests/enable_saml.pp, line: 99)
```

**To Reproduce**
1. make deps
2. aem_port=45652 author_port=45652 make test-integration
   Please note that exit code will be 0 regardless whether there's an error or not for the time being.
   One way to check is if the log doesn't have any occurrence of `has failures`

**Expected behavior**
All tests should pass without failure.

**Environment (please complete the following information if relevant):**
 - AEM 6.5
 - puppet-aem-resources 7.2.3

**Additional context**
The AEM 6.5 we tested against didn't have any service pack installed.
However, we don't expect that installing any service pack would've solved the problem.

**Comment by cliffano** [comment]
Adding a note that the error described in this issue is from the integration test test/integration/manifests/64a_enable_saml_force.pp , and _not_ test/integration/manifests/64_enable_saml.pp

**Comment by mbloch1986** [comment]
That's fixed since version 7.6.0. The issue was that 64a didn't have the force field. I've added it and it's working now.
